Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -32,14 +35,16 @@ enum PackageInfoMapperError: FatalError, Equatable {
         switch self {
         case .noSupportedPlatforms, .unknownByNameDependency, .unknownPlatform, .unknownProductDependency, .unknownProductTarget:
             return .abort
-        case .unsupportedSetting:
+        case .cantFindDefaultPath, .unsupportedSetting:
             return .bug
         }
     }
 
     /// Error description.
     var description: String {
         switch self {
+        case let .cantFindDefaultPath(packageFolder):
+            return "Default path not found for package at \(packageFolder.pathString)."
```

Comment:
```suggestion
            return "Default source path not found for package at \(packageFolder.pathString)."
```

Would that be more clear? Maybe we could even specify the paths we expected? Like:

> `Default source path (Sources/ or Package/) not found for package at \(packageFolder.pathString)`